api/metrics-api-jaxrs/src/main/java/org/hawkular/metrics/api/jaxrs/MetricsServiceLifecycle.java

```diff
@@ -439,6 +439,7 @@ private Session createSession() {
                 .setMaxConnectionsPerHost(HostDistance.REMOTE, newMaxConnections)
                 .setMaxRequestsPerConnection(HostDistance.LOCAL, newMaxRequests)
                 .setMaxRequestsPerConnection(HostDistance.REMOTE, newMaxRequests)
+                .setMaxQueueSize(1024)
```

Comment:
I have hard coded the changed for now simply to see it gets past the error in the perf test. There are a number of driver settings that are worth exposing. Metrics and alerts should probably share the same Session instance.